tokenserver/assignment/memorynode.py

```diff
@@ -20,32 +17,6 @@ def __init__(self, service_entry=None, **kw):
         self._service_entry = service_entry
         self._users = {}
         self._next_uid = 1
-        self._test_settings = {}  # unit test specific overrides
-        self._settings = kw or {}
-
-    @property
-    def settings(self):
-        # Normalize the various settings by picking out the
-        # `tokenserver.` settings from the pyramid settings
-        # registry, remove the namespace prefix so that they
-        # match the values that should be passed in as *kw
-        # to the __init__()
-        settings = dict(
-           map(lambda (k, v): (
-               k.replace('tokenserver.', ''), v),
-               filter(lambda e: e[0].startswith('tokenserver.'),
-                      (get_current_registry().settings or {}).items()))
-        ) or self._settings
-        settings.update(self._test_settings)
-        return settings
-
-    @property
-    def service_entry(self):
-        """Implement this as a property to have the context when looking for
-        the value of the setting"""
```

Comment:
I have no idea why we ever needed to fall back to `get_current_registry` for this property in the memory backend, but removing it didn't cause any tests to fail, so I figure it's some historical legacy that's no longer required.